source/Mission.h

```diff
@@ -67,6 +67,9 @@ class Mission {
 	// Add "never" to the toOffer ConditionSet, preventing this mission from offering.
 	void NeverOffer();
 
+	// Sort order should respect the order field before the name.
+	const bool operator<(const Mission &other) const { return SortHelper(*this, other); }
+
```

Comment:
I don't think it's appropriate for the "natural ordering" of missions to consider the "precedence", especially when the main container ("Set") isn't using that ordering. I think this comparison function should be from `PlayerInfo::CreateMissions`, similar to how the sorting of jobs is not a property of `Mission`.